spec-main/api-app-spec.ts

```diff
@@ -28,33 +24,16 @@ describe('electron module', () => {
   })
 
   describe('require("electron")', () => {
-    let window = null
-
-    beforeEach(() => {
-      window = new BrowserWindow({
-        show: false,
-        width: 400,
-        height: 400,
-        webPreferences: {
-          nodeIntegration: true
-        }
-      })
-    })
-
-    afterEach(() => {
-      return closeWindow(window).then(() => { window = null })
-    })
-
-    it('always returns the internal electron module', (done) => {
-      ipcMain.once('answer', () => done())
-      window.loadFile(path.join(__dirname, 'fixtures', 'api', 'electron-module-app', 'index.html'))
+    it('always returns the internal electron module', () => {
+      require('electron')
```

Comment:
this doesn't assert anything...?

Reply:
Yah the old test didn't either, it just checks that requiring Electron doesn't throw an error